src/Sulu/Bundle/PageBundle/DependencyInjection/SuluPageExtension.php

```diff
@@ -102,6 +102,10 @@ public function prepend(ContainerBuilder $container)
                 [
                     'content' => [
                         'structure' => [
+                            'required_properties' => [
```

Comment:
Shouldn't we add something similar for tags?

Reply:
i have forgotten to add them here - configuration is already implemented